userspace/otpilot/src/main.rs

```diff
@@ -82,21 +84,13 @@ fn run() -> TockResult<()> {
         device_id: [0; 64],
     };
 
-    {
-        let mut idx : usize = 0;
-        for val in "v1.00".as_bytes() {
-            if idx > identity.version.len() { break; }
-            identity.version[idx] = *val;
-            idx = idx + 1;
-        }
-    }
+    let banner = "v1.00".as_bytes();
```

Comment:
Maybe call this `banner_bytes` so as to be parallel with the similar code immediately below?